lib/bom.js

```diff
@@ -309,6 +309,13 @@ declare class Navigator mixins
     // Gecko
     taintEnabled?: () => false;
     oscpu: string;
+
+    // Chrome v0.70+
+    getInstalledRelatedApps?: $ReadOnlyArray<{
+      +platform: string,
+      +id: string
+      +url?: string
```

Comment:
This object should have a version field AFAICT from https://github.com/WICG/get-installed-related-apps/blob/main/EXPLAINER.md

Is that right? Or are there other docs I should be looking at?